apps/simple-trading-app/src/app/components/deal-ticket/deal-ticket-steps.tsx

```diff
@@ -113,18 +143,24 @@ export const DealTicketSteps = ({
     {
       label: t('Choose Position Size'),
       component: (
-        <DealTicketAmount
-          orderType={orderType}
+        <DealTicketSize
           step={step}
-          register={register}
+          min={step}
+          max={maxTrade}
+          onValueChange={onSizeChange}
+          value={parseFloat(orderSize)}
```

Comment:
more safe would be BigNumber or BigInt for this. Here can be a number like 0.0000000008...

Reply:
Currently impossible. We use market.positionDecimal, which is 0 for all.